roles/install-k8s/tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+- name: Disable tx-checksumming on default network interface
```

Comment:
lets add some why we are doing this also a follow up task if any to remove this in the future

Reply:
yes i agree, lets wait until we understand why this is needed for session affinity test to pass

Reply:
To be precise, this is not needed for the feature of session affinity.
This is required for Nodeport service calls that cross the default network interface on ppc64le.